active/0051-strategies.md

```diff
@@ -0,0 +1,248 @@
++ Start Date: 2017-09-07
++ RFC PR: (leave this empty)
+
+# Summary
+
+The goal of this RFC is to solidify the next version of the mechanism that `ember-cli` uses to build final assets. It will allow for a more flexible build pipeline for Ember applications. It also unlocks building experimental features on top. It is a backward compatible change.
+
+# Motivation
+
+The [Packager RFC](https://github.com/chadhietala/rfcs/blob/packager/active/0002-packager.md) submitted by [Chad Hietala](https://github.com/chadhietala) is a little over 2 years old. A lot of things have changed since then and it requires a revision.
+
+The current application build process merges and concatenates input broccoli trees. This behaviour is not well documented and is a tribal knowledge. While the simplicity of this approach is nice, it doesn't allow for extension. We can refactor our build process and provide more flexibility when desired.
+
+Most importantly, it helps us achieve:
```

Comment:
The term `it` is vague in this context, should probably be `Strategies` or something else